app/views/lab_tests/new.html.erb

```diff
@@ -1,7 +1,99 @@
-<h1>New lab test</h1>
+<div class="tw-container tw-mx-auto tw-p-4">
+    <h1 class="tw-text-2xl tw-font-bold tw-mb-4">New <%= params[:test_type]&.titleize || 'Lab' %> Test</h1>
 
-<%= render "form", lab_test: @lab_test %>
+    <% if @lab_test.errors.any? %>
+    <div class="alert alert-danger alert-dismissible fade show tw-mt-4" role="alert">
+        <h2 class="tw-font-bold tw-text-lg">
+            <%= pluralize(@lab_test.errors.count, "error") %> prohibited this test from being saved:
+        </h2>
+        <ul class="tw-list-disc tw-list-inside">
+            <% @lab_test.errors.full_messages.each do |message| %>
+            <li><%= message %></li>
+            <% end %>
+        </ul>
+        <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+    </div>
+    <% elsif flash.present? %>
+    <div class="alert alert-<%= flash.notice.present? ? 'success': 'danger' %> alert-dismissible fade show tw-mt-4" role="alert">
+        <%= flash.notice.present? ? flash.notice : flash.alert %>
+        <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+    </div>
+    <% end %>
 
-<div>
-  <%= link_to "Back to lab tests", lab_tests_path %>
-</div>
+    <%= form_with(model: @lab_test, local: true, class: "tw-space-y-4") do |form| %>
+    <%= form.hidden_field :test_type, value: params[:test_type] %>
+
+    <% if current_user.full_access_roles_can? %>
+    <div class="tw-mb-4">
+        <%= label_tag :user_id, "Select Patient", class: "tw-block tw-text-sm tw-font-medium tw-text-gray-700" %>
+        <%= select_tag :user_id,
+                      options_from_collection_for_select(User.all, :id, :email, params[:user_id]),
```

Comment:
Please read my comment about DB queries in views [here](https://github.com/Code-the-Dream-School/health-keeper/pull/23#discussion_r1889687972).